hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2901,4 +2901,15 @@
       directory deleting service per time interval.
     </description>
   </property>
+
+  <property>
+    <name>ozone.scm.datanode.ratis.volume.free-space.min</name>
+    <value>0MB</value>
```

Comment:
0MB -> 1GB.

Reply:
We can change the default to 1GB but let's add a samller default for unit tests and acceptance tests as we did for container size(Set to 1GB for acceptance tests) so that we don't see any surprises later.

Reply:
Oh, yes, will fix the default value and manually test smaller defaults for related tests, thanks~